src/Stripe.jsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import cx from 'classnames';
+import Bounds from './Bounds';
+
+export const STRIPE_CLASS = 'stripe';
+export const STRIPE_COLLECTION_CLASS = 'stripe--collection';
+export const STRIPE_INVERTED_CLASS = 'stripe--inverted';
+export const STRIPE_HERO_CLASS = 'stripe--hero';
+
+/**
+ * Design System Component: Provides `stripe` styled container for components
+ * @module Stripe
+ */
+class Stripe extends React.Component {
+	render() {
+		const {
+			children,
+			className,
+			backgroundImage,
+			collection,
+			inverted,
+			hero,
+			...other
+		} = this.props;
+
+		const classNames = cx(
+			STRIPE_CLASS,
+			{
+				[STRIPE_COLLECTION_CLASS]: collection,
+				[`${STRIPE_INVERTED_CLASS} inverted`]: inverted,
+				[`${STRIPE_HERO_CLASS} inverted`]: backgroundImage || hero
+			},
+			className
+		);
+
+		const styles = backgroundImage ? {backgroundImage: `url(${backgroundImage})`} : null;
```

Comment:
For MWC/layout components, it's probably better to check if `style` was passed in as prop, and update it gracefully if so.